canonical-websites/www.ubuntu.com
# 1609 cloud juju

## Done

* update /cloud/juju to vbt
* replaced 3 broken charms and a few broken bundles with new ones from this issue #1888
* pointed to JAAS beta

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [page](http://0.0.0.0:8001/cloud/juju)
- compare to the [design](https://www.dropbox.com/work/00_web_team/_ubuntu.com/new%20projects/vanilla%20update/Patterns%20inventory/2.%20Cloud?preview=cloud-juju.png)


## Issue / Card

Fixes #1609
Fixes #1888
